### DISCUSSION TITLE
Number of installs in Analytics are all 0

### DISCUSSION THREAD

**Discussion by hatoo** [Homebrew/discussions#4610] [Everyday usage] [answered]
### Output of `brew config`

```shell
...
```

### Output of `brew doctor`

```shell
...
```

### Description of issue

Related to #4548

https://formulae.brew.sh/formula/peco
https://formulae.brew.sh/formula/hey
https://formulae.brew.sh/formula/wrk

Some packages have Analytics that all numbers are 0.
I remember those packages had some positive numbers of installs before.
It's hard to believe those packages are not installed within one year.

![image](https://github.com/Homebrew/discussions/assets/1667746/1973a895-1b86-4386-9b0d-008c978c43dd)

**Comment by SMillerDev** [comment]
The data for the year is incomplete, but it could be that nobody who had analytics enabled installed this.

**Comment by ackdav** [comment, marked as answer]
Noticed the same pattern - the numbers for [https://formulae.brew.sh/formula/localstack](https://formulae.brew.sh/formula/localstack) are all at 0. However, it's actively being downloaded and used (just downloaded it yesterday myself with analytics enabled). Any ideas on how we can resolve this? Happy to look into it myself, but since I'm not familiar with the codebase, ideas on where to look would be very helpful. 
edit: nevermind, it fixed itself :pray:
